CHANGELOG.rst

```diff
@@ -1,6 +1,48 @@
 [Unreleased]
 ------------
 
+Added
+^^^^^
+* `@senwu`_: Support CSV, TSV, Text input data format.
+  For CSV format, ``CSVDocPreprocessor`` treats each line in the input file as
+  a document. It assumes that each column is one section and content in each column
+  as one paragraph as defalt. However, if the column is complex, an advanced parser
+  may be used by specifying ``parser_rule`` parameter in a dict format where key
+  is the column index and value is the specific parser.
+
+.. note:
+    In Fonduer v0.4.2, you can use ``CSVDocPreprocessor``:
```

Comment:
This will be a breaking change, right? We no longer have the CSVPathsPreprocessor? This will require a bump to `v0.5.0`

Reply:
No, I removed ``CSVPathsPreprocessor`` since it has weird semantic.